* Use the more lightweight py-poetry-core.

* Avoid the installation of the test suite into the top of Python's site-lib.

* Relax some version requirements and set an upper bound for py-hier-config
  because the 3.x release contains breaking changes.

--- pyproject.toml.orig	2024-11-22 10:15:59 UTC
+++ pyproject.toml
@@ -2,9 +2,8 @@
 
 [build-system]
 # See -> PEP 517
-requires = ["poetry>=1.5.1", "poetry-core>=1.6.1", "wheel>=0.38.4", "packaging>=23.0"]
+requires = ["poetry-core>=1.6.1", "wheel>=0.38.4", "packaging>=23.0"]
 build-backend = "poetry.core.masonry.api"
-requires-python = ">=3.8.0"
 
 ##############################################################################
 #
@@ -38,7 +37,6 @@ packages = [
 packages = [
     # Syntax -> https://python-poetry.org/docs/pyproject/#packages
     { include = "ciscoconfparse/*" },
-    { include = "tests/*" },
 ]
 # See -> https://python-poetry.org/docs/pyproject/#include-and-exclude
 include = [
@@ -80,9 +78,9 @@ dnspython = "^2.4.2"
 python = ">=3.8,<4.0.0"
 passlib = "^1.7.4"
 dnspython = "^2.4.2"
-loguru = "0.7.2"
+loguru = ">=0.7.2"
 toml = ">=0.10.2"
-hier_config = "2.2.3"
+hier_config = ">=2.2.3,<3"
 deprecated = ">=1.2.14"
 
 [tool.poetry.urls]
